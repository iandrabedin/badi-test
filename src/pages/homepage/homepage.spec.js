import React from "react";
import { render, cleanup } from "@testing-library/react";
import Homepage from "./homepage";

afterEach(cleanup);

describe("Homepage", () => {
  it("should render as expected", () => {
    const { asFragment } = render(<Homepage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
