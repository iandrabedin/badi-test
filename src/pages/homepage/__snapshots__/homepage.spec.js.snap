// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage should render as expected 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <h1>
      Recipe Puppy is an ingredient based recipe search
    </h1>
    <div
      class="row"
    />
  </div>
</DocumentFragment>
`;
